spec/Section 5 -- Validation.md

```diff
@@ -446,7 +446,7 @@ SameResponseShape(fieldA, fieldB):
 - If {typeA} or {typeB} is Scalar or Enum:
   - If {typeA} and {typeB} are the same type return {true}, otherwise return
     {false}.
-- Assert: {typeA} and {typeB} are both composite types.
+- Assert: {typeA} and {typeB} are both Object, Interface or Union types.
```

Comment:
This reads ambiguously; it could be interpretted as `(typeA is object and typeB is object) or (typeA is union and typeB is union) or (typeA is an interface and typeB is an interface)` rather than `(typeA is object or union or interface) and (typeB is object or union or interface)`. Perhaps:

```suggestion
- Assert: {typeA} is an object, union or interface type.
- Assert: {typeB} is an object, union or interface type.
```